quickstartKotlin/src/main/java/com/twilio/video/quickstart/kotlin/VideoActivity.kt

```diff
@@ -633,40 +637,41 @@ class VideoActivity : AppCompatActivity() {
-                .roomName(roomName)
-
-        /*
-         * Add local audio track to connect options to share with participants.
-         */
-        localAudioTrack?.let { connectOptionsBuilder.audioTracks(listOf(it)) }
-
-        /*
-         * Add local video track to connect options to share with participants.
-         */
-        localVideoTrack?.let { connectOptionsBuilder.videoTracks(listOf(it)) }
+        val connectionOptions = createConnectOptions(accessToken) {
+            roomName(roomName)
+            /*
+             * Add local audio track to connect options to share with participants.
+             */
+            audioTracks(listOf(localAudioTrack))
+            /*
+             * Add local video track to connect options to share with participants.
+             */
+            videoTracks(listOf(localVideoTrack))
 
-        /*
-         * Set the preferred audio and video codec for media.
-         */
-        connectOptionsBuilder.preferAudioCodecs(listOf(audioCodec))
-        connectOptionsBuilder.preferVideoCodecs(listOf(videoCodec))
+            /*
+             * Set the preferred audio and video codec for media.
+             */
+            preferAudioCodecs(listOf(audioCodec))
+            preferVideoCodecs(listOf(videoCodec))
 
-        /*
-         * Set the sender side encoding parameters.
-         */
-        connectOptionsBuilder.encodingParameters(encodingParameters)
+            /*
+             * Set the sender side encoding parameters.
+             */
+            encodingParameters(encodingParameters)
 
-        /*
-         * Toggles automatic track subscription. If set to false, the LocalParticipant will receive
-         * notifications of track publish events, but will not automatically subscribe to them. If
-         * set to true, the LocalParticipant will automatically subscribe to tracks as they are
-         * published. If unset, the default is true. Note: This feature is only available for Group
-         * Rooms. Toggling the flag in a P2P room does not modify subscription behavior.
-         */
-        connectOptionsBuilder.enableAutomaticSubscription(enableAutomaticSubscription)
+            /*
+             * Toggles automatic track subscription. If set to false, the LocalParticipant will receive
+             * notifications of track publish events, but will not automatically subscribe to them. If
+             * set to true, the LocalParticipant will automatically subscribe to tracks as they are
+             * published. If unset, the default is true. Note: This feature is only available for Group
+             * Rooms. Toggling the flag in a P2P room does not modify subscription behavior.
+             */
+            enableAutomaticSubscription(enableAutomaticSubscription)
+        }
 
-        room = Video.connect(this, connectOptionsBuilder.build(), roomListener)
+        room = connect(this, accessToken, roomListener) {
+            connectionOptions
```

Comment:
Let's us the Video.kt [connect function](https://github.com/twilio/twilio-video-android/blob/b57e0611dfd1bbf7b7d40d24f06610014c8ebebb/video/ktx/src/main/java/com/twilio/video/ktx/Video.kt#L79) to take advantage of the [ConnectOptionsBuilder](https://github.com/twilio/twilio-video-android/blob/b57e0611dfd1bbf7b7d40d24f06610014c8ebebb/video/ktx/src/main/java/com/twilio/video/ktx/ConnectOptions.kt#L6) typealias.